### PR TITLE
Grid-Group By Update

### PR DESCRIPTION
**Pull request by IGSvt**
- Updated image reference
- Updated keywords
- Updated definitions

### CODE REVIEW

File: en/images/grid_group_by_demo.png
Comment (imincheva) [inline en/images/grid_group_by_demo.png:1]: There's excess space underneath the grid, which is transparent.

File: en/images/grid_group_by_demo@2x.png
Comment (imincheva) [inline en/images/grid_group_by_demo@2x.png:1]: There's excess space underneath the grid, which is transparent.

**Review by imincheva — CHANGES_REQUESTED** [review]
I suggested one change on the text and left comments about the images.

File: en/components/grid-grouping.md [hunk 1]
```diff
@@ -14,11 +14,11 @@ Use Grid Group By to organize data records in a hierarchical collection of group
...
 
 ## GroupBy Row
 
-The Grid has a `GroupBy Row` symbol under `Features` that represents a special row in the Grid that is like a heading preceding the grouped records with some summary information about their amount within the group. Adding this symbol has only a visual effect on your design to make it more realistic. To switch the GroupBy feature on, insert and configure a `Group Area`.
+The Grid has a `GroupBy Row` symbol/component under `Grid Features` that represents a special row in the Grid that is like a heading preceding the grouped records with some summary information about their amount within the group. Adding this symbol/component has only a visual effect on your design to make it more realistic. To switch the GroupBy feature on in Sketch, insert and configure a `Group Area`. In Figma and Adobe Xd you can simply insert the `GroupBy Row` component.
```
[the remark this answers is not in the document]
Comment (IGSvt) [reply en/components/grid-grouping.md:21]: You are right

**Review by imincheva — APPROVED** [review]

**Review by sdimchevski — APPROVED** [review]

**Pull request merged**